package.json

```
@@ -97,41 +121,70 @@
     "@types/node": "^24.3.1",
     "@typescript-eslint/eslint-plugin": "8.43.0",
     "@typescript-eslint/parser": "8.43.0",
     "@typescript-eslint/utils": "8.43.0",
     "angular-eslint": "^20.2.0",
+    "@types/node": "^24.3.1",
+    "@typescript-eslint/eslint-plugin": "8.43.0",
+    "@typescript-eslint/parser": "8.43.0",
+    "@typescript-eslint/utils": "8.43.0",
+    "angular-eslint": "^20.2.0",
     "autoprefixer": "^10.4.21",
     "electron": "38.0.0",
```

Biome (2.1.2)

[error] The key electron was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)